File: JavaScript/Bits/getbit.js
/**
 * @description Récupère le i-ème bit de n.
 * @param {number} n - Le nombre à traiter
 * @param {number} i - L'index du bit à récupérer
 * @return {number} 0 ou 1
 */
const getBit = (n, i) => (n >> i) & 1;

// Tests
console.log(getBit(2, 0)); // 0
console.log(getBit(2, 1)); // 1

File: Python/Bits/get_bit.py
def get_bit(n: int, i: int) -> int:
    """
    Description: 
        Récupère le i-ème bit de n.
    
    Paramètres:
        n: {int} -- Le nombre à traiter
        i: {int} -- L'index du bit à récupérer

    Retourne:
        {int} -- 0 ou 1

    Exemple:
        >>> get_bit(2, 0)
        0

        >>> get_bit(2, 1)
        1    
    """
    return (n >> i) & 1